bin/plot_krun_results

```diff
@@ -682,9 +682,9 @@ class ProcessExecChart(object):
                                         + int(one_pc * 2.0)))
             # Clamp the extent of the inset.
             if inset_xlimit[1] < INSET_MIN_ITERS:
-                inset_xlimit = (0, min(INSET_MIN_ITERS, self.x_bounds[1] - self.x_bounds[0]))
+                inset_xlimit = (0, min(INSET_MIN_ITERS, self.x_bounds[1] - self.x_bounds[0] - 1))
```

Comment:
In the common case for our data, `self.x_bounds[0]` will be 0 and `self.x_bounds[1]` will be 2000, so your change should be fine because the minimum will always be `INSET_MIN_ITERS`. 

In the case where the user has passed in `--xlimits` and `self.xlimits[1] - self.xlimits[0] < 50` then the minimum won't be `INSET_MIN_ITERS`, so this is the case where we need to check the off-by-one. 

I don't fully understand what the test data looks like, but if there is no data for this pexec, then `self.xbounds == (0, 0)` and so your `-1` will make `inset_xlimit == (0, -1)`, which seems wrong to me. But maybe that isn't the test case?